docs/hu/migration/migration270to280.adoc

```diff
@@ -0,0 +1,25 @@
+= v2.7.0 → v2.8.0
+
+coff:ee v2.7.0 -> v2.8.0 migrációs leírás, újdonságok, változások leírása
+
+== Global
+
+=== coffee-jpa
+
+💥 ***BREAKING CHANGE*** 💥
+
+A `hu.icellmobilsoft.coffee.jpa.sql.batch.BatchService` átalakítások:
+
+* A `BatchService` Hibernate függősége megszünt (a `coffee-jpa`-nak továbbra is függősége a Hibernate).
+* Típusa osztályból interface lett.
+* Ős interface az új `hu.icellmobilsoft.coffee.jpa.sql.batch.IJpaBatchService`.
+* Implementációja a Frapp:EE projekt `frappee-hibernate` modulba lett átmozgatva.
+
+==== Átállás
+
+A `hu.icellmobilsoft.coffee.jpa.sql.batch.BatchService` használata:
+
+* Be kell behúznunk a `hu.icellmobilsoft.frappee:frappee-hibernate` modult a függőségek közé és készen is vagyunk. Itt
+ található a `hu.icellmobilsoft.frappee.batch.hibernate.HibernateBatchService` implementáció.
+* Ha kiterjesztettük a korábbi `BatchService`-t, akkor a `HibernateBatchService`-t kell kiterjesztenünk.
+Bővebb információ a Frapp:EE dokumentációban található.
```

Comment:
Linket ide is szerintem majd pls :)

Reply:
https://github.com/i-Cell-Mobilsoft-Open-Source/coffee/issues/662 -ben lesz megoldva.